code/game/machinery/computer/dna_console.dm

```diff
@@ -36,11 +36,21 @@
 	var/combine
 	var/radduration = 2
 	var/radstrength = 1
+	///the max instability of the advanced injector.
+	var/advanced_injector_limit = 50
 	var/max_chromosomes = 6
-
+	///Amount of mutations we can store
 	var/list/buffer[NUMBER_OF_BUFFERS]
+	///mutations we have stored
 	var/list/stored_mutations = list()
+	///chromosomes we have stored
 	var/list/stored_chromosomes = list()
+	///combinations of injectors for the 'injector selection'. format is list("Elsa" = list(Cryokinesis, Geladikinesis), "The Hulk" = list(Hulk, Gigantism), etc) Glowy and the gang being an initialized datum
+	var/list/injector_selection = list()
+	///max amount of selections you can make
+	var/max_injector_instability = 2
```

Comment:
I feel like the names on these are reversed?

Reply:
I have absolutely no idea how that happened, I distinctly remember not doing this